core/core.py

```diff
@@ -98,114 +99,13 @@ def get_process(input_dict: dict) -> str:
-                    "category": "API",
-                    "description": "API für das Frontend",
-                    "creation_timestamp": "20200219...",
-                    "last_timestamp": "20200219...",
-                    "metrics": {
-                        "codelines": 20000,
-                        "admins": 10,
-                        "recovery_time": 5
-                    }
-                },
-                {
-                    "uid": "b141f94973a43cf8ee972e9dffc1b004",
-                    "weight": 2,
-                    "name": "Hadoop Cluster",
-                    "category": "Datenbank",
-                    "description": "Big Data Plattform",
-                    "creation_timestamp": "20200219...",
-                    "last_timestamp": "20200219...",
-                    "metrics": {
-                        "codelines": 20000,
-                        "admins": 10,
-                        "recovery_time": 5
-                    }
-                }
-            ]
-        },
-        "score": 80,  # percent as integer
-        "actual_target_metrics": {
-            "codelines": {
-                "actual": {
-                    "average": 30,
-                    "max": 50,
-                    "min": 20,
-                    "standard_deviation": 5,
-                    "total": 300,  # summe
-                },
-                "target": {
-                    "average": 30,
-                    "total": 300,  # summe
-                },
-                "component_count": 10,
-                "fulfillment": True
-            },  # true means that the metric is fulfilled --> no problem.
-            "admins": {
-                "actual": {
-                    "average": 30,
-                    "max": 50,
-                    "min": 20,
-                    "standard_deviation": 5,
-                    "total": 300,     # summe
-                },
-                "target": {
-                    "average": 30,
-                    "total": 300,     # summe
-                },
-                "component_count": 10,
-                "fulfillment": True
-            },
-            "recovery_time": {
-                "actual": {
-                    "average": 0.65,
-                    "max": 0.8,
-                    "min": 0.5,
-                    "standard_deviation": 5,
-                    "total": 0.4,     # summe
-                },
-                "target": {
-                    "average": 0.8,
-                    "total": 0.64,     # summe
-                },
-                "component_count": 2,
-                "fulfillment": True
-            },  # false means that the metric is not fulfilled --> problem.
-        }
-    }
+    metrics_dict = metric_handler.get_metrics_data()
+
+    output_dict = processing.calculations.start_calculate_risk(process_dict, metrics_dict)
+
+    output_json = processing.dict_to_json(output_dict)
     
-    return processing.dict_to_json(data)
```

Comment:
Finde hier die alte Version eigentlich schöner, aber ich glaube das ist Geschmacksache oder gibt es dafür eine Guideline.

Reply:
In core wird überall 'return ouput_json' verwendet. Daher haben wir es auch übernommen.